### ISSUE TITLE
Crash

### ISSUE THREAD

**Issue by tosbaha** [KyoheiG3/SpringIndicator#11]
Sometimes I get crash report related to SpringIndicator from users via Crashlytics. I can't reproduce on my end.

``` swift
        if let timer = sender as? NSTimer, key = timer.userInfo as? String where key == Me.ContractAnimationKey {
            let timer = createStrokeTimer(timeInterval: strokeDuration * 2, userInfo: Me.GroupAnimationKey, repeats: true)

            setStrokeTimer(timer)
        }
```

```
Thread : Crashed: SpringIndicator.Timer.Thread
0  CoreFoundation                 0x1829aa7d8 -[__NSCFTimer userInfo] + 28
1  MyApp                     0x10022fabc specialized SpringIndicator.onStrokeTimer(AnyObject) -> () (SpringIndicator.swift:299)
2  MyApp                     0x10022ca94 @objc SpringIndicator.onStrokeTimer(AnyObject) -> () (SpringIndicator.swift)
3  Foundation                     0x18398ce6c __NSFireTimer + 88
4  CoreFoundation                 0x182a00c3c __CFRUNLOOP_IS_CALLING_OUT_TO_A_TIMER_CALLBACK_FUNCTION__ + 28
5  CoreFoundation                 0x182a008e0 __CFRunLoopDoTimer + 884
6  CoreFoundation                 0x1829fdff4 __CFRunLoopRun + 1520
7  CoreFoundation                 0x18292cca0 CFRunLoopRunSpecific + 384
8  Foundation                     0x1838a52bc -[NSRunLoop(NSRunLoop) runMode:beforeDate:] + 308
9  Foundation                     0x1838fa8f4 -[NSRunLoop(NSRunLoop) run] + 88
10 libdispatch.dylib              0x197e196e8 _dispatch_call_block_and_release + 24
```

**Comment by ashrat07** [comment]
@tosbaha Did you find any solution for this problem? I am also facing this problem quite frequently.

**Comment by tosbaha** [comment]
@ashrat07 I fixed the bug. You can check #14 to see the bug fix.

**Comment by KyoheiG3** [comment]
Fixed in [2.0.0](https://github.com/KyoheiG3/SpringIndicator/releases/tag/2.0.0)
